### DISCUSSION TITLE
Is it possible to set dagster.yaml config options in the code location

### DISCUSSION THREAD

**Discussion by the4thamigo-uk** [dagster-io/dagster#19119] [Q&A]
Hi, we are deploying a code location to kubernetes using dagster-user-deployments. I have various jobs in the code location which are tagged for concurrency control using `run_queue` -> `tag_concurrency_limits`. When working locally I set these limits in dagster.yaml. However, it is not clear how I set such options in the kubernetes deployment. In particular, I would really like to be able to somehow configure these settings directly in my code location rather than having to update and redeploy the main dagster instance when I add a new tag, is this possible?

**Comment by andrescolodrero** [comment]
Good question!. It is also not working for me. I want to change the storage location for all the jobs like:

local_artifact_storage:
  module: dagster.core.storage.root
  class: LocalArtifactStorage
  config:
    base_dir: "/dagster_home/storage"

I added "dagster.yml" to the root directory of the project
/etc/dagster/app/dagster.yml
etc/dagster/app/my_app (the folder contains the assest)

But i dont see that code locations are getting any of this config.

**Reply by the4thamigo-uk** [reply]
I got this working with the main webserver helm chart by using the setting [here](https://github.com/dagster-io/dagster/blob/master/helm/dagster/values.yaml#L1058). The chart basically uses many of these values to construct the `dagster.yaml` in a configmap in kubernetes. There is a section where you can add other stuff that isnt dealt with through explicit helm values [here](https://github.com/dagster-io/dagster/blob/master/helm/dagster/values.yaml#L1326).

It seems like dagster.yaml is only relevant to the main webserver, not the code locations as such.  It sould be nice though if I could add certain configs directly in the code location somehow, either through python annotations or settings in the `Definitions`, but maybe this isnt possible?...
